org.eclipse.jdt.core/pom.xml

```diff
@@ -17,7 +17,7 @@
     <version>4.28.0-SNAPSHOT</version>
   </parent>
   <artifactId>org.eclipse.jdt.core</artifactId>
-  <version>3.33.100-SNAPSHOT</version>
+  <version>3.33.101-SNAPSHOT</version>
```

Comment:
@snjeza, @gayanper : please next time don't increment bundle version **if it is already incremented** for the current release and if not, it shouldn't be +1 but +100.

The strategy is to increment minor segment by +1 if there is a new API and service segment by +100 if this is the **first** change in the release.